Prevent arrow keys from scrolling page

stevenjb opened the pull request:
Actual tested fix for issue #161 to prevent keys from scrolling the page in paper-slider.

keanulee commented:
Change looks good. There aren't currently tests for increment/decrement keyboard shortcuts; would you mind adding some and testing for `event.defaultPrevented` as well?

stevenjb commented:
OK, I added some tests and verifed that the defaultPrevented check fails without my change but passes with the change. Hopefully I updated the pull request successfully?

keanulee commented:
LGTM. Thanks @stevenjb
